projects/plugins/jetpack/_inc/client/main.jsx

```diff
@@ -295,13 +295,12 @@ class Main extends React.Component {
 			 * parameter that JITMs will set (showCouponRedemption=true), and show the screen only
 			 * when the user came from a a JITM.
 			 */
-			if ( ! this.props.isSiteConnected || forceShow ) {
+			if ( ! this.props.connectionStatus.hasConnectedOwner || forceShow ) {
```

Comment:
Wouldn't this be problematic? If I connect my site, but not my user, I cannot access Jetpack settings or the Jetpack dashboard anymore; the partner banner takes over all the time and I cannot change any settings anymore.